docs/reference/pyboard.py.rst

```diff
@@ -59,6 +60,14 @@ with the device.::
     $ pyboard.py --device /dev/ttyACM0 -c 'print(1+1)'
     2
 
+If you are often interacting with the same device, you can set the environment
+variable ``PYBOARD_DEVICE`` as an alternative to using the ``--device``
+command line option.  For example, the following is equivalent to the previous
```

Comment:
This should mention `PYBOARD_BAUD` too

Reply:
Thanks, good catch. I've updated the docs.